components/Heroes/BenchmarkTable.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+
+import style from './heroes.css';
+
+
+const metrics = [
+  'percentile',
+  'gold_per_min',
+  'xp_per_min',
+  'last_hits_per_min',
+  'kills_per_min',
+  'hero_damage_per_min',
+  'hero_healing_per_min',
+  'tower_damage_per_min',
+];
+const roundFn = (number) => Math.round(number * 100) / 100;
+const alignCenter = { textAlign: 'center' };
+
+const BenchmarkRow = ({ data, id, percentile }) => (
+  <TableRow selectable={false} className={style[`BenchmarkRow-${id % 2}`]}>
+    {metrics.map((metric, i) => {
+      if (metric === 'percentile') {
+        return (
+          <TableRowColumn key={i} style={alignCenter} className={style.BenchmarkColNo}>
+            {parseFloat(percentile) * 100}%
+          </TableRowColumn>
+        );
+      }
+
+      return (
+        <TableRowColumn key={i} style={alignCenter}>
+          {roundFn(parseFloat(data[metric][id].value))}
+        </TableRowColumn>
+      );
+    })}
+  </TableRow>
+);
+
+export default ({ data }) => (
+  <Table wrapperStyle={{ overflow: 'visible' }} className={style.BenchmarkTable}>
+    <TableHeader displaySelectAll={false} adjustForCheckbox={false}>
+      <TableRow>
+        <TableHeaderColumn tooltip="Percentile" style={alignCenter} className={style.BenchmarkColNo}>%</TableHeaderColumn>
```

Comment:
I think it only needs to be in the UI repo.  I don't see a use case where the server would need display strings.  

The reason they are in dotaconstants now is that the Jade templates (on server side) require the strings as well as the UI.  However we should be able to retire those with the SPA and therefore move tooltips.json from dotaconstants to here as well.

You may as well start a new string file for any strings that are UI-only, and we'll merge them with the ones from tooltips.json when we don't need them on the server side anymore.